### ISSUE TITLE
Main Knob seems inoperative - no random seq and calibration issue

### ISSUE THREAD

**Issue by bluasango** [TomWhitwell/TuringMachine#141]
Hi!
I'm an absolute newbie to modular and diy (super combo). The turing machine is my 3rd diy project. The ones before worked flawlessly.
I think the main (big) knob is not working:

1) During calibration (acc. to the vimeo video) I get a constant frequency, but as I turn the main knob to either position, the tone stays stable.
2) When giving it a slow clock, and putting the OUT to 1V/OCT of my Mini Mod or my XR22 it creates a random sequence, but that stays locked all the time irrespective of the main knob position.

I'm using a clock from my DAW, I know one shouldn't, but I don't have a different one yet.
I am glad for any advice! 

![IMG_6513](https://user-images.githubusercontent.com/71961938/94359418-5fb26f80-00a7-11eb-8547-8112b9b44c97.jpg)
![IMG_6508](https://user-images.githubusercontent.com/71961938/94359421-60e39c80-00a7-11eb-8c14-c81f8597b7c5.jpg)
![IMG_6511](https://user-images.githubusercontent.com/71961938/94359424-64772380-00a7-11eb-9aca-3560b1ccea84.jpg)
![IMG_6507](https://user-images.githubusercontent.com/71961938/94359431-6e992200-00a7-11eb-82ec-030e5b5b5ef6.jpg)

**Comment by bluasango** [comment]
https://vimeo.com/462294903
&
https://vimeo.com/462294894

**Comment by TomWhitwell** [comment]
Are you getting noise from the noise output?

**Comment by bluasango** [comment]
Hi Tom, thanks for that quick reply.

No! Didn't check that before.

**Comment by bluasango** [comment]
no noise

**Comment by TomWhitwell** [comment]
Ok that’s the problem - check the around the transistor 2n3904 and components around that, look for missing solder points or shorts around there, reflow anything that looks dodgy - also on the tl074 pins

**Comment by bluasango** [comment]
Alright, will do so in the next hours and give an update then. Thanks!

**Comment by bluasango** [comment]
I resoldered every single joint and replaced the 2n3904 with a spare one. Still the same problems. Still no noise out.
Can it have any other reason?

**Comment by TomWhitwell** [comment]
Tweak the trimmer, because that is the gain for the noise generator - if it's turned all the way down you might not be able to hear it. 

This is the schematic for the Noise generator 


I know they're hard to read, but you can see it's a path from the transistor, through two gates on the TL074 op amp. That chip seems to be OK, because it's the same chip used in the clock input and the CV output, both of which appear to be working. 
 
![image](https://user-images.githubusercontent.com/1890544/94371008-24d22b00-00eb-11eb-8498-9a6687e9d778.png)

**Comment by bluasango** [comment]
Thank you. I'll look into that!

**Comment by bluasango** [comment]
Yes. I moved the trimmer slightly right and there's noise right away! 
But it seems to not lock any sequence, it's only random. Also the sequence length is ignored now I think.

**Comment by bluasango** [comment]
Sequence length is fine, sorry.

**Comment by TomWhitwell** [comment]
Have you done this: https://vimeo.com/163160088

**Comment by bluasango** [comment]
Yes several times. There is a mini threshold, where the sequence jumps from locked to random while tweaking the trimmer, irrespective of the main knob position. Could it be that simply the pot (main knob) is defective?

**Comment by TomWhitwell** [comment]
Faulty pot is possible but not very likely. Do you have an offset in your system - something that can output +5 to -5 ish volts (like the middle channels on a Maths)? If so, you can put that into the CV input and see if the system works fine like that.

**Comment by biospherix** [comment]
Really glad I came across this thread. I was having a very similar issue and noted on the schematic that the Electrolytic 10uF in C6 was on the noise path. I suspected this was the problem, changed it and now the unit (and Volts expansion) work perfectly. However, I couldn't find any small pot capacitors and had the settle with a tall pot bent over but it still works. Thanks. 

![IMG_5646](https://user-images.githubusercontent.com/84015584/123509385-c6e96880-d6a7-11eb-8a82-de71354eb903.jpg)
